build.py

```diff
@@ -2338,6 +2348,23 @@ def enable_all():
         required=False,
         help="Override specified backend CMake argument in the build as <backend>:<name>=<value>. The argument is passed to CMake as -D<name>=<value>. This flag only impacts CMake arguments that are used by build.py. To unconditionally add a CMake argument to the backend build use --extra-backend-cmake-arg.",
     )
+    parser.add_argument(
+        "--split-build",
+        action="store_true",
+        required=False,
+        help="Split the intermediate build artifacts into independently build-able targets",
+    )
+    parser.add_argument(
+        "--cache-from-map",
+        action="append",
+        required=False,
+        help="A set of docker images to `--cache-from` where applicable to help speedup builds",
+        default=[
+            "tritonserver_buildbase",
+            "tritonserver_buildbase_cache0",
+            "tritonserver_buildbase_cache1",
+        ],
+    )
```

Comment:
I understand that it was called a `map` before, but it's really just a list. What do you think about changing this to something like `--cache-image` or `--cache-images`? We can make use of the multiple arguments in a couple of different ways. We can use the `action="append"` or the `nargs='+'` fields ([SO comment for example](https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse)). I might lean towards the `--cache-image` with `action="append"` API just because we have precedent.

Reply:
I think name was used to  keep consistency with original command which is relates to: `docker build --cache-from` 
```
$ docker build --help | grep cache-from
      --cache-from stringArray        External cache sources (e.g., "user/app:cache", "type=local,src=path/to/dir")
```

Reply:
How about `--cache-from-list`?